packages/flutter_tools/bin/xcode_backend.sh

```diff
@@ -101,6 +101,8 @@ BuildApp() {
   local preview_dart_2_flag=""
   if [[ -n "$PREVIEW_DART_2" ]]; then
     preview_dart_2_flag="--preview-dart-2"
+  else
+    preview_dart_2_flag="--no-preview-dart-2"
```

Comment:
Due to the commit message I thought that this CL switches `--preview-dart-2` on by-default. Now here the `else` branch is the default which **switches it explicitly off** by-default. Why is that?

Reply:
The option is passed by flutter tools from the outside. This shell script is never used on its own.